mbtiles/src/mbtiles.rs

```diff
@@ -849,4 +1027,122 @@ mod tests {
         assert!(matches!(result, Err(MbtError::AggHashMismatch(..))));
         Ok(())
     }
+
+    #[actix_rt::test]
+    async fn meter_to_lnglat() {
+        let cases = [
+            [
+                -20037508.34,
+                -20037508.34,
+                -179.99999997494382,
+                -85.05112877764508,
+            ],
+            [
+                20037508.34,
+                20037508.34,
+                179.99999997494382,
+                85.05112877764508,
+            ],
+            [0.0, 0.0, 0.0, 0.0],
+            [3000.0, 9000.0, 0.026949458523585643, 0.08084834874097371],
+        ];
+        for case in cases.iter() {
+            let actual = webmercator_to_wgs84(case[0], case[1]);
+            let exp_lng = case[2];
+            let exp_lat = case[3];
+            assert_relative_eq!(actual[0], exp_lng, epsilon = 1e-4);
+            assert_relative_eq!(actual[1], exp_lat, epsilon = 1e-4);
+        }
```

Comment:
probably not worth it, keep it simple like this:

```suggestion
        let (lng, lat) = webmercator_to_wgs84(-20037508.34, -20037508.34);
        assert_relative_eq!(lng, -179.99999997494382, epsilon = f64::EPSILON);
        assert_relative_eq!(lat, -85.05112877764508, epsilon = f64::EPSILON);
        
        let (lng, lat) = webmercator_to_wgs84(20037508.34, 20037508.34);
        assert_relative_eq!(lng, 179.99999997494382, epsilon = f64::EPSILON);
        assert_relative_eq!(lat, 85.05112877764508, epsilon = f64::EPSILON);
        
        let (lng, lat) = webmercator_to_wgs84(0.0, 0.0);
        assert_relative_eq!(lng, 0.0, epsilon = f64::EPSILON);
        assert_relative_eq!(lat, 0.0, epsilon = f64::EPSILON);
        
        let (lng, lat) = webmercator_to_wgs84(3000.0, 9000.0);
        assert_relative_eq!(lng, 0.026949458523585643, epsilon = f64::EPSILON);
        assert_relative_eq!(lat, 0.08084834874097371, epsilon = f64::EPSILON);
```

Reply:
Simpler and easy to read!